src/apis/vaults.ts

```diff
@@ -209,6 +212,20 @@ export class DefaultVaultsAPI {
         return new UInt(new TypeRegistry(), 0) as PolkaBTC;
     }
 
+    private async getSecureCollateralThreshold(): Promise<u128> {
+        return await this.api.query.vaultRegistry.secureCollateralThreshold();
+    }
+
+    async getIssuablePolkaBTC(): Promise<string> {
+        const collateral = new DefaultCollateralAPI(this.api);
+        const totalLockedDot = await collateral.totalLockedDOT();
+        const oracle = new DefaultOracleAPI(this.api);
+        const exchangeRate = await oracle.getExchangeRate();
+        const exchangeRateU128 = this.api.createType("u128", exchangeRate);
```

Comment:
Why create a type via this call? If you need a `Big` like type, better to use the `Big` constructor like `new Big(exchangeRate)`. The polkadot type convert has practically no error handling, unfortuently.

Reply:
I'm using `createType` because this is the approach recommended by the documentation. `createType` ultimately calls `createTypeUnsafe` function, which handles errors as described in the header comments: https://github.com/polkadot-js/api/blob/master/packages/types/src/create/createType.ts#L62